client/src/app/api/rest.ts

```diff
@@ -227,50 +161,20 @@ export const getJobFunctions = (): AxiosPromise<JobFunction[]> => {
   return APIClient.get(`${JOB_FUNCTIONS}`, jsonHeaders);
 };
 
-// Tag categories
-
-export const getTagCategories = (): AxiosPromise<Array<TagCategory>> => {
```

Comment:
Can you move this work to a separate PR?

Reply:
@ibolton336, not really, working with `Ref` took me to make `id` not an option and this triggered a cascade of changes on the involved queries, which I aligned using axios as this is how we handle them.

Reply:
Seems like some good changes here. Thanks.